app/src/main/java/com/github/meeplemeet/MainActivity.kt

```
@@ -18,6 +31,59 @@ class MainActivity : ComponentActivity() {
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContent { AppTheme { Surface(modifier = Modifier.fillMaxSize()) { Text("Hello World") } } }
+
+    // Set up dependencies for AuthViewModel
+    val firestore = FirebaseFirestore.getInstance()
+    val auth = FirebaseAuth.getInstance()
+
+    // Configure Firebase emulators for testing - only if available
+    try {
+      Log.d("MainActivity", "Attempting to configure Firebase emulators...")
+
+      // Try to connect to emulators - this will throw if they're not running
+      auth.useEmulator("10.0.2.2", 9099)
+      firestore.useEmulator("10.0.2.2", 8080)
+
+      Log.d("MainActivity", "Firebase emulators configured successfully")
+    } catch (e: IllegalStateException) {
+      // Emulators already configured - this is normal
+      Log.d("MainActivity", "Firebase emulators already configured")
```

You should `throw e` instead of logging it